windows/wsl_distro_install_uninstall/unregister_wsl_distro.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+# Unregister and uninstall a WSL distribution
+
+- include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "wsl --unregister {{ wsl_distribution_name }} | Out-File {{ wsl_file_path_win }}\\unregister_distro.txt"
```

Comment:
Suggest changing file name to `wsl_unregister_distro.txt', so that it can be align with the others.

Reply:
Ok. Thanks.